src/dlgRoomExits.cpp

```diff
@@ -1335,7 +1335,7 @@ void dlgRoomExits::slot_out_textEdited(const QString& text)
 void dlgRoomExits::slot_stub_nw_stateChanged(int state)
 {
     if (state == Qt::Checked) {
-        if (mpHost->mpMap->mpRoomDB->getRoom(nw->text().toInt()) != 0) {
+        if (mpHost->mpMap->mpRoomDB->getRoom(nw->text().toInt()) != nullptr) {
```

Comment:
And more simplifications:

    if (mpHost->mpMap->mpRoomDB->getRoom(nw->text().toInt())) {